pytorch_lightning/accelerators/accelerator.py

```diff
@@ -66,10 +66,10 @@ def train_or_test(self):
             results = self.trainer.train()
         return results
 
-    def batch_to_device(self, batch: Any, device: torch.device):
+    def batch_to_device(self, batch: Any, device: Optional[torch.device] = None):
         model = self.trainer.get_model()
         if model is not None:
-            return model.transfer_batch_to_device(batch, device)
```

Comment:
this doesn't look backward compatible.

cc @Borda @tchaton

Reply:
@tchaton ^^